test/unit/org/apache/cassandra/service/paxos/uncommitted/PaxosStateTrackerTest.java

```diff
@@ -206,7 +175,8 @@ public void manualRebuild() throws Throwable
         SystemKeyspace.savePaxosWritePromise(dk(0), cfm1, ballots[2]);
         SystemKeyspace.savePaxosProposal(commit(cfm1, 2, ballots[3]));
 
-        try (SystemProp forceRebuild = SystemProp.set(PaxosStateTracker.FORCE_REBUILD_PROP, true))
+        Boolean prev = FORCE_PAXOS_STATE_REBUILD.setBoolean(true);
```

Comment:
Replace it with `WithProperties`, note that this applies to all the places which match this scenario

Reply:
I agree here, as new changes need to use `WithProperties`, so I have fixed it. But still, as I understand it, the goal is to move to `CassandraRelevantProperties`, so as we're changing ~200 files I'd like to avoid any kind of refactoring - this will simplify the commit investigation for others. 

So the question here is - can we create another issue to do the changes you are mentioning?